pkg/apis/edgefs.rook.io/v1/devices.go

```diff
@@ -74,13 +80,15 @@ type CcowTenant struct {
 	SyncPut          int `json:"sync_put,omitempty"`
 	SyncPutNamed     int `json:"sync_put_named"`
 	CommitWait       int `json:"sync_put_commit_wait"`
+	ChunkSize        int `json:"chunkmap_chunk_size,omitempty"`
 }
 
 type CcowNetwork struct {
 	BrokerInterfaces string `json:"broker_interfaces"`
 	ServerUnixSocket string `json:"server_unix_socket"`
 	BrokerIP4addr    string `json:"broker_ip4addr,omitempty"`
 	ServerIP4addr    string `json:"server_ip4addr,omitempty"`
+	NoIP4Frag        int    `json:"no_ipfrag,omitempty"`
```

Comment:
we probably should document this as well.

Reply:
Added to the doc